presto-parser/src/main/java/com/facebook/presto/sql/tree/GroupingSets.java

```diff
@@ -47,7 +48,7 @@ private GroupingSets(Optional<NodeLocation> location, List<List<QualifiedName>>
         super(location);
         requireNonNull(sets, "sets is null");
         checkArgument(!sets.isEmpty(), "grouping sets cannot be empty");
-        this.sets = ImmutableList.copyOf(sets.stream().map(ImmutableList::copyOf).collect(Collectors.toList()));
+        this.sets = sets.stream().map(ImmutableList::copyOf).collect(toImmutableList());
```

Comment:
if you would put into immutableCollectors then maybe you could use it here without calling map function